backend/entities/academics/section_entity.py

```diff
@@ -54,6 +54,13 @@ class SectionEntity(EntityBase):
         back_populates="section"
     )
 
+    # Relationship subset of members queries for non-students
+    staff: Mapped[list["SectionMemberEntity"]] = relationship(
+        back_populates="section",
+        viewonly=True,
+        primaryjoin="and_(SectionEntity.id==SectionMemberEntity.section_id, SectionMemberEntity.member_role!='STUDENT')",
```

Comment:
Ah, so this was the solution!!! Very interesting - thanks for including this @KrisJordan !

Reply:
Might as well use in those instances, too, since the solution is the same and it is straightforward. It's too bad this gets jammed into a string and evaled rather than being straight source from the start, but I was going off SQLAlchemy example